From 4e7189147e7f2f8171fbc0d77e163851a802ccdb Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 6 May 2019 21:22:21 -0600
Subject: [PATCH] [SQUASH][DNM] Merge tag 'android-9.0.0_r37' into
 staging/lineage-16.0_merge-android-9.0.0_r37

Android 9.0.0 Release 37 (PQ3A.190505.002)

* tag 'android-9.0.0_r37':
  Import translations. DO NOT MERGE
  Import translations. DO NOT MERGE

https://github.com/haggertk/android_packages_apps_KeyChain/tree/lineage-16.0_merge-android-9.0.0_r37

Change-Id: I4ca6229d2a2276030c79be98094efa26aedb71a0
---
 res/values-en-rXC/strings.xml | 18 +++++++++---------
 res/values-mr/strings.xml     |  2 +-
 2 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/res/values-en-rXC/strings.xml b/res/values-en-rXC/strings.xml
index b75e2e9..0780068 100644
--- a/res/values-en-rXC/strings.xml
+++ b/res/values-en-rXC/strings.xml
@@ -16,13 +16,13 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_name" msgid="170210454004696382">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‎‏‎‏‎‎‏‎‏‏‏‎‎‏‎‏‏‎‏‎‏‏‎‎‎‎‎‎‏‏‎‎‏‎‎‎‎‎‎‎‎‎‎‏‎‎‎‎‎‎‏‎‏‎‎‏‏‏‏‏‎‎Key Chain‎‏‎‎‏‎"</string>
-    <string name="title_no_certs" msgid="8350009443064722873">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‏‏‎‎‏‏‏‏‏‎‎‎‎‏‎‎‏‏‎‎‎‏‎‏‎‏‏‎‎‏‎‎‏‎‎‏‎‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‏‏‎‏‏‏‎‎‏‎No certificates found‎‏‎‎‏‎"</string>
-    <string name="title_select_cert" msgid="3588447616418041699">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‏‎‎‎‏‏‏‎‎‏‏‎‎‏‎‏‏‏‎‏‎‎‎‎‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‎‎‏‏‏‎‎‏‏‏‏‎‏‏‎‎‎‏‏‎Choose certificate‎‏‎‎‏‎"</string>
-    <string name="requesting_application" msgid="1589142627467598421">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‏‏‎‏‏‎‎‎‎‎‏‏‎‏‏‏‎‎‎‏‎‎‏‏‎‎‏‏‎‎‏‏‎‎‏‏‏‏‏‎‏‎‎‎‏‏‎‎‎‏‏‎‏‎‎‏‎‏‎‏‎‏‎The app %s has requested a certificate. Choosing a certificate will let the app use this identity with servers now and in the future.‎‏‎‎‏‎"</string>
-    <string name="requesting_server" msgid="5832565605998634370">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‏‎‎‎‏‎‏‏‎‏‏‏‏‏‎‎‎‎‏‎‎‎‎‏‏‎‎‏‎‏‎‏‏‎‏‎‏‏‎‏‎‎‏‎‏‏‎‎‎‎‎‏‎‎The app has identified the requesting server as %s, but you should only give the app access to the certificate if you trust the app.‎‏‎‎‏‎"</string>
-    <string name="install_new_cert_message" msgid="4451971501142085495">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‏‏‏‎‏‏‏‎‎‏‎‎‎‏‎‎‏‎‏‎‎‎‏‏‏‏‏‏‎‎‎‏‎‎‎‏‏‏‎‏‏‏‏‎‎‏‎‎‎‏‏‏‏‎‏‏‏‎‏‏‏‎You can install certificates from a PKCS#12 file with a %1$s or a %2$s extension located in external storage.‎‏‎‎‏‎"</string>
-    <string name="install_new_cert_button_label" msgid="903474285774077171">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‎‏‏‎‎‏‎‎‎‏‎‎‏‏‏‎‎‏‎‎‏‎‎‎‏‎‎‏‏‎‏‏‏‏‎‏‏‎‏‎‎‎‎‏‏‎‏‏‏‏‏‎‎‏‏‏‏‎‎‏‏‎Install certificate‎‏‎‎‏‎"</string>
-    <string name="allow_button" msgid="3030990695030371561">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‎‏‎‏‎‎‎‎‏‎‎‎‎‎‎‏‏‏‏‎‏‏‏‏‏‏‎‏‎‎‏‏‎‏‏‏‎‏‏‎‎‏‏‎‎‏‎‏‏‎‏‎‎‏‏‏‎‏‎‎‏‎Select‎‏‎‎‏‎"</string>
-    <string name="deny_button" msgid="3766539809121892584">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‏‎‏‎‎‎‏‎‎‎‏‎‏‎‏‏‏‎‎‎‎‎‎‎‎‎‎‎‎‎‎‎‏‏‎‏‏‎‎‎‏‎‎‏‏‏‏‏‏‏‎‎‎‏‏‏‎‏‎‎‎‎Deny‎‏‎‎‏‎"</string>
+    <string name="app_name" msgid="170210454004696382">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‎‏‎‏‎‎‏‎‏‏‏‎‎‏‎‏‏‎‏‎‏‏‎‎‎‎‎‎‏‏‎‎‏‎‎‎‎‎‎‎‎‎‎‏‎‎‎‎‎‎‏‎‏‎‎‏‏‏‏‏‎‎Key Chain‎‏‎‎‏‎"</string>
+    <string name="title_no_certs" msgid="8350009443064722873">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‏‏‎‎‏‏‏‏‏‎‎‎‎‏‎‎‏‏‎‎‎‏‎‏‎‏‏‎‎‏‎‎‏‎‎‏‎‏‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‏‏‎‏‏‏‎‎‏‎No certificates found‎‏‎‎‏‎"</string>
+    <string name="title_select_cert" msgid="3588447616418041699">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‏‎‎‎‏‏‏‎‎‏‏‎‎‏‎‏‏‏‎‏‎‎‎‎‏‏‎‎‎‎‎‏‏‏‏‏‏‏‎‏‏‎‎‎‏‏‏‎‎‏‏‏‏‎‏‏‎‎‎‏‏‎Choose certificate‎‏‎‎‏‎"</string>
+    <string name="requesting_application" msgid="1589142627467598421">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‏‏‎‏‏‎‎‎‎‎‏‏‎‏‏‏‎‎‎‏‎‎‏‏‎‎‏‏‎‎‏‏‎‎‏‏‏‏‏‎‏‎‎‎‏‏‎‎‎‏‏‎‏‎‎‏‎‏‎‏‎‏‎The app %s has requested a certificate. Choosing a certificate will let the app use this identity with servers now and in the future.‎‏‎‎‏‎"</string>
+    <string name="requesting_server" msgid="5832565605998634370">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‎‏‎‎‎‎‏‏‏‏‎‎‎‏‎‏‏‎‏‏‏‏‏‎‎‎‎‏‎‎‎‎‏‏‎‎‏‎‏‎‏‏‎‏‎‏‏‎‏‎‎‏‎‏‏‎‎‎‎‎‏‎‎The app has identified the requesting server as %s, but you should only give the app access to the certificate if you trust the app.‎‏‎‎‏‎"</string>
+    <string name="install_new_cert_message" msgid="4451971501142085495">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‏‏‏‎‏‏‏‎‎‏‎‎‎‏‎‎‏‎‏‎‎‎‏‏‏‏‏‏‎‎‎‏‎‎‎‏‏‏‎‏‏‏‏‎‎‏‎‎‎‏‏‏‏‎‏‏‏‎‏‏‏‎You can install certificates from a PKCS#12 file with a %1$s or a %2$s extension located in external storage.‎‏‎‎‏‎"</string>
+    <string name="install_new_cert_button_label" msgid="903474285774077171">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‎‏‏‎‎‏‎‎‎‏‎‎‏‏‏‎‎‏‎‎‏‎‎‎‏‎‎‏‏‎‏‏‏‏‎‏‏‎‏‎‎‎‎‏‏‎‏‏‏‏‏‎‎‏‏‏‏‎‎‏‏‎Install certificate‎‏‎‎‏‎"</string>
+    <string name="allow_button" msgid="3030990695030371561">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‎‏‎‏‎‎‎‎‏‎‎‎‎‎‎‏‏‏‏‎‏‏‏‏‏‏‎‏‎‎‏‏‎‏‏‏‎‏‏‎‎‏‏‎‎‏‎‏‏‎‏‎‎‏‏‏‎‏‎‎‏‎Select‎‏‎‎‏‎"</string>
+    <string name="deny_button" msgid="3766539809121892584">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‏‎‏‎‎‎‏‎‎‎‏‎‏‎‏‏‏‎‎‎‎‎‎‎‎‎‎‎‎‎‎‎‏‏‎‏‏‎‎‎‏‎‎‏‏‏‏‏‏‏‎‎‎‏‏‏‎‏‎‎‎‎Deny‎‏‎‎‏‎"</string>
 </resources>
diff --git a/res/values-mr/strings.xml b/res/values-mr/strings.xml
index 82aca0e..39ec831 100644
--- a/res/values-mr/strings.xml
+++ b/res/values-mr/strings.xml
@@ -19,7 +19,7 @@
     <string name="app_name" msgid="170210454004696382">"कि चेन"</string>
     <string name="title_no_certs" msgid="8350009443064722873">"कोणतीही प्रमाणपत्रे आढळली नाहीत"</string>
     <string name="title_select_cert" msgid="3588447616418041699">"प्रमाणपत्र निवडा"</string>
-    <string name="requesting_application" msgid="1589142627467598421">"%s अॅप एका प्रमाणपत्राची विनंती करत आहे. एक प्रमाणपत्र निवडणे या अॅपला या सर्व्हरसह आता आणि भविष्यात ही ओळख वापरू देईल."</string>
+    <string name="requesting_application" msgid="1589142627467598421">"%s अ‍ॅप एका प्रमाणपत्राची विनंती करत आहे. एक प्रमाणपत्र निवडणे या अ‍ॅपला या सर्व्हरसह आता आणि भविष्यात ही ओळख वापरू देईल."</string>
     <string name="requesting_server" msgid="5832565605998634370">"अॅपने विनंती केलेल्या सर्व्हरला %s म्हणून ओळखले आहे, परंतु तुमचा अॅपवर विश्वास असेल तरच तुम्ही अॅपला प्रमाणपत्रामध्ये प्रवेश द्यावा."</string>
     <string name="install_new_cert_message" msgid="4451971501142085495">"तुम्ही अतिरिक्त स्टोरेजमध्ये स्थित %1$s किंवा %2$s विस्तारासह एका PKCS#12 फाइलवरुन प्रमाणपत्र इंस्टॉल करु शकता."</string>
     <string name="install_new_cert_button_label" msgid="903474285774077171">"प्रमाणपत्र इंस्टॉल करा"</string>
-- 
2.17.1

